From 8700a6545bd30a2e8ec27fa17c9f99c5ac743c9d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Sat, 30 Jul 2011 22:53:31 +0800
Subject: [PATCH] Revert "mmc: Add status IRQ and status callback function to
 mmc platform data"

This reverts commit cfd429c65523ddf1be48ff1835a2776d93688fb2.
---
 include/linux/amba/mmci.h | 2 --
 1 file changed, 2 deletions(-)

diff --git a/include/linux/amba/mmci.h b/include/linux/amba/mmci.h
index ee58a21b6075..6b4241748dda 100644
--- a/include/linux/amba/mmci.h
+++ b/include/linux/amba/mmci.h
@@ -13,8 +13,6 @@ struct mmci_platform_data {
 	int	gpio_wp;
 	int	gpio_cd;
 	unsigned long capabilities;
-	unsigned int status_irq;
-	int (*register_status_notify)(void (*callback)(int card_present, void *dev_id), void *dev_id);
 };
 
 #endif
-- 
2.35.3

